scripts/docs/generate_color_palette.py

```diff
@@ -0,0 +1,268 @@
+from typing import Tuple
+
+import matplotlib.colors as mcolors
+import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle
+from PIL import ImageColor
```

Comment:
Would be good to include a module docstring which says that this script is used to generate the color palette figures for the documentation

Reply:
Changed.